src/components/footer/Footer.module.css

```diff
@@ -0,0 +1,8 @@
+.icon:hover {
+  transform: scale(1.1);
+}
+
+.termsLink {
```

Comment:
Can we leverage the Chakra style attributes on the `termsLink` element? This will also help to remove the need for this extra stylesheet!

Reply:
Good shout